content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md

```diff
@@ -37,11 +37,19 @@ Click the timezone dropdown to select a timezone to use for the dashboard. Selec
 
 Click the refresh button (**{{< icon "refresh" >}}**) to manually refresh the dashboard's data.
 
-#### Refresh a single dashboard cell
+#### Manually refresh a single dashboard cell
 
 1. Click the **{{< icon "gear" >}}** on the dashboard cell you want to refresh.
 2. Click **{{< icon "refresh" >}} Refresh**.
 
+### Automatically refresh dashboard
+
+Click **Enable Auto Refresh**. In the window that appears, configure the following options:
+  - **Until**: Select **Indefinite** to automatically refresh continuously. Select **Custom** to set a time to stop automatically refreshing.
+  - **Inactivity Timeout**: Set how long the dashboard should be inactive for to time out and stop automatically refreshing.
```

Comment:
```suggestion
  - **Inactivity Timeout**: Set how long the dashboard should be inactive before timeout stops automatically refreshing.
```

Reply:
is this change accurate?

Reply:
Maybe its worth saying "user should be inactive" or "page should be inactive" to be clearer about what is the trigger for the timeout?

Reply:
How about `"Set how long the dashboard should be inactive before the auto-refresh times out and stops automatically refreshing" ` ?

Reply:
"the dashboard should be inactive" is the part im not sure about whether the word should be dashboard or user

Reply:
yeah sorry I didn't see your comment because I didn't refresh github